db.go

```diff
@@ -20,10 +20,10 @@ import (
 )
 
 const (
-	fileLockName       = "FLOCK"
-	dataFileNameSuffix = ".SEG"
-	hintFileNameSuffix = ".HINT"
-	mergeFinNameSuffix = ".MERGEFIN"
+	fileLockName        = "FLOCK"
+	dataFileNameSuffix  = ".SEG"
+	hintFileNameSuffix  = ".HINT"
+	mergeFileNameSuffix = ".MERGEFIN"
```

Comment:
thanks.

Fin is not a typo, it means `Finished`.

Reply:
Indeed, you are correct, `Fin` is excellent. I shall disregard this.